collect-logs

```diff
@@ -540,12 +575,13 @@ def bundle_logs(tmpdir, tarfile, extrafiles=[]):
     call(args)
 
 
-def get_juju(binary_path, model=DEFAULT_MODEL, cfgdir=None, inner=False):
+def get_juju(binary_path, model=DEFAULT_MODEL, cfgdir=None, inner=False,
+             juju_ssh=False):
```

Comment:
nitpick. I think it makes more sense to default `juju_ssh=True` here and then specify it as False when calling `get_juju()` in `"__main__"`